srcpkgs/kubectx/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'kubectx'
+pkgname=kubectx
+version=0.9.4
+revision=1
+makedepends="go"
+depends="fzf kubectl"
+short_desc="Faster way to switch between clusters and namespaces in kubectl"
+maintainer="fanyx <fanyx@fanyx.xyz>"
+license="Apache-2.0"
+homepage="https://github.com/ahmetb/kubectx"
+distfiles="https://github.com/ahmetb/kubectx/archive/v${version}.tar.gz"
+checksum=91e6b2e0501bc581f006322d621adad928ea3bd3d8df6612334804b93efd258c
+nopie=y
+
+do_build() {
+	go build -o ${worksrc}/bin/kubectx ./cmd/kubectx/
+	go build -o ${worksrc}/bin/kubens ./cmd/kubens/
+}
```

Comment:
use the go build style instead

Reply:
i was gonna but it won't work as the repo root doesn't hold a main.go
code is in subdirectories, but those dont hold go.mod files either

Reply:
Look at other templates, this is usually done by setting go_package and go_import_path

Reply:
ah, go_package was the one i was missing
let me add that